src/main/kotlin/uk/gov/justice/digital/hmpps/esupervisionapi/offender/OffenderCheckinRepository.kt

```diff
@@ -52,8 +53,15 @@ open class OffenderCheckin(
   @JdbcTypeCode(SqlTypes.JSON)
   open var surveyResponse: Map<String, Object>?,
 
+  /**
+   * Will hold the latest status and/or error of any sent notifications.
+   */
+  @Column("notifications", nullable = true)
```

Comment:
We don't need to track notification status yet. Ideally we'd define a new table rather than storing JSON on the offender directly.